portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +412,67 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script type="text/javascript">
+
+	var dropdown = document.querySelector('.pagination-items-per-page');
```

Comment:
You could add an `id` with the namespace instead of searching for this class, this may cause side effects.

Reply:
Fixed